website/docs/reference/datasources/airtable.md

```diff
@@ -4,399 +4,334 @@ sidebar_position: 1
+Use the **Delete A Record** command to delete existing records in your spreadsheet.
 
-![Getting BaseID](/img/baseIDInAPIDocumentation.jpeg)
+After you fill in fields for **Base ID** and **Table Name**, you just need to provide a record's `id` in the **Record ID** field. Each object should have an `id` key with the `id` of the record you are updating, and a `fields` key containing an object with the record's new data:
 
-#### Table Name
+```javascript
+// in the Record field
+recdo2NkX3ucpnOLb
+```
 
-Open your database to get the table name; it is in the top-left corner. For example, the table name for the image below is "_Tasks_."
+When record is deleted successfully, the response includes the `id` of the deleted record.
 
-![Getting Table Name](/img/airtable_doc_3.png)
+---
 
-#### Records
+**Example**: delete a record from a table `users`.
 
-A [record ](https://support.airtable.com/hc/en-us/articles/360021333094-Getting-started-tables-records-and-fields)is the base equivalent of a row in a spreadsheet. Each record is an item in a list. For example, in a table of books, each record is a different book. Unlike a spreadsheet, a record in a mobile Airtable base appears as a tappable card.
+**Setup**: create your query called `DeleteUser` based on your Airtable datasource. This query should use the **Delete A Record** command. Enter the appropriate values for **Base ID** and **Table Name**. Finally, You should have a [Table widget](/reference/widgets/table) `UsersTable` containing your spreadsheet data from a [**List Records**](#list-records) query.
 
-#### Record ID
+Create a [Button widget](/reference/widgets/button) on the canvas and update its **Label** to "Delete." Set its **onClick** event to execute your `DeleteUser` query:
 
-Within Airtable, each record has a unique identifier known as a Record ID. If you are familiar with Entity-Relationship Diagrams or ERDs, then the record id would be the primary key. For most, record IDs remain behind the scenes.
+```javascript
+// in the Delete button's onClick event
+{{ DeleteUser.run() }}
+```
 
-After expanding a record, take notice of your web address bar. The URL will be as below**:**
+To delete a record, you need only provide the `id` of the record to delete. In your `DeleteUser` query's **Record ID** field, bind the value of the selected row of the `UsersTable`:
 
-```
-airtable.com/appxxxxxxxxx/tblxxxxxxxxx/viwxxxxxxxx/recxxxxxxxx
+```javascript
+// DeleteUser query's Row Index field
+{{ UsersTable.selectedRow.id }}
 ```
 
-The `recxxxxxxx` portion of the URL string is the unique record id.
+Now when the button is clicked, the query is run and the corresponding row is deleted from your Airtable base.
 
-```
-Example URL - https://airtable.com/appUgxxxxx/tblxZhrixxxxx/viwGM7xxxxx/recw3xxxxx?blocks=hide
+## Commands
```

Comment:
why do we have a table of the commands at the bottom after explaining all the queries?